gradebookng/tool/pom.xml

```diff
@@ -117,12 +117,10 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-core</artifactId>
-			<scope>provided</scope>

```

Comment:
Is this no longer provided?

Reply:
It's declared as provided in master already just removing the override here.